index.js

```diff
@@ -22,18 +22,25 @@ export type ResolveOptions = {
 };
 */
 
-function getImportSources(filePath, parserOpts) {
-  let file = loadFileSync(filePath, parserOpts);
-  let importSources = [];
+function getFileExtension(filename)
+{
+  var ext = /^.+\.([^.]+)$/.exec(filename);
+  return ext == null ? "" : ext[1];
```

Comment:
There's already a method in Node for this: https://nodejs.org/api/path.html#path_path_extname_path

Reply:
cool - I did not know that!